server/services/homekit/lib/createBridge.js

```diff
@@ -35,6 +37,8 @@ async function createBridge() {
     .map((device) => this.buildAccessory(device))
     .filter((accessory) => accessory !== null);
 
+  this.gladys.event.on(EVENTS.TRIGGERS.CHECK, eventFunctionWrapper(this.notifyChange.bind(this, accessories)));
```

Comment:
I just re-read the code and found this, it seems to me that every time a home kit bridge is created, it creates a subscription on `this.gladys.event.on(EVENTS.TRIGGERS.CHECK`. It'll create multiple subscription, no ?

Reply:
Indeed, it will create multiple subscriptions 🤔 for example when we reload the bridge after adding new accessory to Gladys. Thanks to delay before notify it should send only 1 message but need to improve that.
I quickly check `lib/event` file not sure how to delete previous one before create the new one. Any idea?

Reply:
We need to implement `removeListener()` in `Event`?

Reply:
Indeed, we miss a removeListener!

Reply:
https://nodejs.org/api/events.html#emitterremovelistenereventname-listener

Reply:
Done! I added `removeListener` on our Event implementation and use it for HomeKit reload